src/groundstation.py

```diff
@@ -110,28 +108,25 @@ def receive(self):
                                                              libcsp.conn_sport(conn),
                                                              libcsp.conn_dst(conn),
                                                              libcsp.conn_dport(conn)))
-
             while True:
                 # Read all packets on the connection
                 packet = libcsp.read(conn, 100)
                 if packet is None:
                     break
 
-                if libcsp.conn_dport(conn) == 10:
+                if libcsp.conn_dport(conn) == 1:
```

Comment:
Is this check still necessary, since we're receiving commands from multiple different ports? I have since removed it on my branch so I'd like to know what you think.

Reply:
Not necessary actully. This may be only useful for testing hk service haha. I push this to github housekeeping branch just to save lastest version so that we could compare and refer to easier when rewritting our own ground software in the future.

Reply:
OK, fair enough!